File: app/src/main/java/com/chemecador/secretaria/requests/NoteRequest.java
package com.chemecador.secretaria.requests;

import com.chemecador.secretaria.items.Note;


public class NoteRequest {

    private String title;
    private String content;
    private int status;


    public NoteRequest(Note note) {
        this.title = note.getTitle();
        this.content = note.getContent();
        this.status = note.getStatus();
    }
}

File: app/src/main/java/com/chemecador/secretaria/requests/NoteRequest.kt
package com.chemecador.secretaria.requests

import com.chemecador.secretaria.items.Note

class NoteRequest(note: Note) {
    private val title: String?
    private val content: String?
    private val status: Int

    init {
        title = note.title
        content = note.content
        status = note.status
    }
}